astropy/stats/sigma_clipping.py

```diff
@@ -157,19 +157,33 @@ def _sigma_clip(data, sigma=3, sigma_lower=None, sigma_upper=None, iters=1,
     Note that along the other axis, no points would be masked, as the
     variance is higher.
     """
+    def perform_clip(_filtered_data, _kwargs):
```

Comment:
Do we have rules on how variables in internal functions should be handled?   And should this hold up merging this in?